### PR TITLE
adding info about replace upgrades and inplace upgrades per osdocs-4180

### PR DESCRIPTION
**Pull request by lahinson**
Related OCP doc Jira: https://issues.redhat.com/browse/OSDOCS-4180

This PR adds information about replace upgrades and in-place upgrades for hosted control planes.

I requested review from Yu Qi Zhang in the related Jira. I did not find him in GitHub to add as a reviewer.

### CODE REVIEW

**Review by yuqi-zhang — COMMENTED** [review]
Generally looks fine, thanks!

**Review by oafischer — APPROVED** [review]
Looks good, thanks!

File: multicluster_engine/hosted_control_planes/hosted_control_planes_configure.adoc [hunk 1]
```diff
@@ -348,9 +348,11 @@ spec:
           - <zone-2>
 ----
 
-Replace `zone-1` with the name of the zone in the region.
+Replace `zone-1` with the name of the zone in the region. Replace `zone-2` with the name of the zone in the region. 
 
-Replace `zone-2` with the name of the zone in the region. 
+In the previous example, the `upgradeType` is `Replace`. To upgrade a hosted cluster, you can use a replace upgrade or an in-place upgrade. A replace upgrade creates instances in the new version while it removes old instances from the previous version. This upgrade type is a good choice in cloud environments where this level of immutability is cost effective. In-place upgrades directly update the operating systems of the instances. This type is a good choice for environments where the infrastructure constraints are higher, such as bare metal.
```
Comment (swopebe) [inline multicluster_engine/hosted_control_planes/hosted_control_planes_configure.adoc]: For clarity, we may want to break out the upgrade types so user can easily find def. and term?

A _replace upgrade_ creates instances in the new version while it removes old instances from the previous version. This upgrade type is a good choice in cloud environments where this level of immutability is cost effective. 

An _in-place_ upgrade directly updates the operating systems of the instances. This type is a good choice for environments where the infrastructure constraints are higher, such as bare metal.

(parallel and off set in two paragraphs to draw user to the term. We could also do a DL but I think with just these two, not necessary.)

thoughts?

Comment (lahinson) [reply multicluster_engine/hosted_control_planes/hosted_control_planes_configure.adoc]: Great idea. I can make those changes. Thanks!

**Comment by lahinson** [comment]
@swopebe I just committed the changes that you suggested. Thanks!

**Comment by swopebe** [comment]
With Olivers approval, the merge is ready for you when you are. 

@oafischer since you want to update the readme, this may be a good tidbit:

_Link to your Jira or GitHub issue in the PR. You can merge after your PR is reviewed by a peer and a developer._ (though it looks like Laura has the dev. discussion in the JIRA so that works, too).

**Comment by lahinson** [comment]
Thanks @swopebe and @oafischer!

**Pull request merged**